content/writing_tests/anatomy.md

```diff
@@ -14,17 +14,49 @@ This guide assumes that you have finished the "Getting started" tutorial or that
 The minimum setup for a Sakuli project looks like this:
 
 - <i class="fas fa-folder"></i> **your-sakuli-project** *Home folder of the project*
-  - <i class="far fa-file"></i> **package.json** *All dependencies and setups for Node.js*
-  - <i class="far fa-file"></i> **sakuli.properties** *(optional: Configuration for all testsuites)*
+  - <i class="fas fa-folder"></i> **_logs** *An automatically generated folder for error screenshots and sakuli.log*
+    - <i class="fas fa-folder"></i> **_screenshots** *Folder for error screenshots*
+    - <i class="far fa-file"></i> **sakuli.log** *Logs of Sakuli execution*
```

Comment:
Not indented correctly when website is build. `_screenshots` and `sakuli.log` should be elements of `_logs`.

![image](https://user-images.githubusercontent.com/25754312/93740418-b8d35c80-fbea-11ea-8b1a-7f1deb594593.png)


In addition, `_logs` is created in every test suite not in the project itself.
![image](https://user-images.githubusercontent.com/25754312/93740568-123b8b80-fbeb-11ea-974a-78fce0daf734.png)

Reply:
That's strange. For me, it is rendered correctly:
![image](https://user-images.githubusercontent.com/48590838/93745266-89751d80-fbf3-11ea-9b60-e7a6dfed7a55.png)